src/DssSpellCollateral.sol

```diff
@@ -32,92 +32,61 @@ contract DssSpellCollateralAction {
+        //  Add RETH-A as a new Vault Type
+        //  Poll Link: [https://vote.makerdao.com/polling/QmfMswF2]
+
+        DssExecLib.addNewCollateral(
+            CollateralOpts({
+                ilk:                   "RETH-A",
+                gem:                   RETH,
+                join:                  MCD_JOIN_RETH_A,
+                clip:                  MCD_CLIP_RETH_A,
+                calc:                  MCD_CLIP_CALC_RETH_A,
+                pip:                   PIP_RETH,
+                isLiquidatable:        true,
+                isOSM:                 true,
+                whitelistOSM:          true,
+                ilkDebtCeiling:        5 * MILLION,
+                minVaultAmount:        15 * THOUSAND,                // debt floor - dust in DAI
+                maxLiquidationAmount:  2 * MILLION,
+                liquidationPenalty:    13_00,                        // 13% penalty on liquidation
+                ilkStabilityFee:       ONE_FIVE_PCT_RATE,            // 1.50% stability fee
+                startingPriceFactor:   110_00,                       // Auction price begins at 110% of oracle price
+                breakerTolerance:      50_00,                        // Allows for a 50% hourly price drop before disabling liquidation
+                auctionDuration:       120 minutes,
+                permittedDrop:         45_00,                        // 45% price drop before reset
+                liquidationRatio:      170_00,                       // 170% collateralization
+                kprFlatReward:         250,                          // 250 DAI tip - flat fee per kpr
+                kprPctReward:          10                            // 10% chip - per kpr
+            })
+        );
 
+        DssExecLib.setStairstepExponentialDecrease(MCD_CLIP_CALC_RETH_A, 90 seconds, 99_00);
+        DssExecLib.setIlkAutoLineParameters("RETH-A", 5 * MILLION, 3 * MILLION, 8 hours);
         // ChainLog Updates
-        // DssExecLib.setChangelogAddress("XXX", XXX);
-        // DssExecLib.setChangelogAddress("PIP_XXX", PIP_XXX);
-        // DssExecLib.setChangelogAddress("MCD_JOIN_XXX_A", MCD_JOIN_XXX_A);
-        // DssExecLib.setChangelogAddress("MCD_CLIP_XXX_A", MCD_CLIP_XXX_A);
-        // DssExecLib.setChangelogAddress("MCD_CLIP_CALC_XXX_A", MCD_CLIP_CALC_XXX_A);
-    }
+        // Add the new join, clip, and abacus to the Chainlog
+        DssExecLib.setChangelogAddress("RETH",                 RETH);
+        DssExecLib.setChangelogAddress("PIP_RETH",             PIP_RETH);
+        DssExecLib.setChangelogAddress("MCD_JOIN_RETH_A",      MCD_JOIN_RETH_A);
+        DssExecLib.setChangelogAddress("MCD_CLIP_RETH_A",      MCD_CLIP_RETH_A);
+        DssExecLib.setChangelogAddress("MCD_CLIP_CALC_RETH_A", MCD_CLIP_CALC_RETH_A);
 
-    function updateCollaterals() internal {
-        // ------------------------------- Collateral updates -------------------------------
+        // Rely Median to Oracles team
```

Comment:
No need to get merged with this PR I assume because we aren't doing oracles this spell anymore?